.github/workflows/release.yml

```
@@ -276,14 +276,46 @@ jobs:
 
-          # 4. Build the .msix package (unsigned — signing is a separate step)
+          # 4. Build the .msix package
           $msixPath = "amux-${{ matrix.target }}.msix"
           & $makeAppx pack /d $msixDir /p $msixPath /o
           Write-Host "Built: $msixPath"
 
           # Surface for upload
           Add-Content -Path $env:GITHUB_ENV -Value "MSIX_PATH=$msixPath"
 
+      - name: Sign MSIX — Azure Trusted Signing
+        if: runner.os == 'Windows' && secrets.AZURE_TENANT_ID != ''
```

actionlint (1.7.12)

[error] context "secrets" is not allowed here. available contexts are "env", "github", "inputs", "job", "matrix", "needs", "runner", "steps", "strategy", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)